boards/hail/chip_layout.ld

```diff
@@ -7,3 +8,13 @@ RAM_ORIGIN  = 0x20000000;
 RAM_LENGTH  = 0x00020000;
 
 MPU_MIN_ALIGN = 8K;
+
+STACK_SIZE = 0x1000;
+
+MEMORY
+{
+  rom (rx)  : ORIGIN = 0x00010000, LENGTH = 0x00020000
+  prog (rx) : ORIGIN = 0x00030000, LENGTH = 0x00040000
+  ccfg (rx) : ORIGIN = 0,          LENGTH = 0
```

Comment:
I'm a little wary of the copy/paste that this introduces here. i.e. the `ccfg` section doesn't really make sense to be in the hail memory map at all? Granted it was there in the general case, but that's hidden away. Now that it's in the "hail-specific" file, it's odd to see it.

Reply:
Yeah that can be removed from boards that do not use it.

Reply:
If you remove the `ccfg` definition here, does this part of the linker file cause problems?

```
    .ccfg : {
        KEEP(*(.ccfg))
    } > ccfg
```

Or does it just silently ignore undefined, empty things?

Reply:
Doesn't seem to mind.